spec/controllers/ahoy/visits_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe Ahoy::VisitsController, :type => :controller do
+  routes { Ahoy::Engine.routes }
+
+  it "creates visits" do
+    begin
+      previous_value = Ahoy.track_visits_immediately = true
```

Comment:
You may want to do this in two steps if your intent is to preserve the old value.  This code is setting both values to true.